Lib/gftools/utils.py

```diff
@@ -239,3 +240,191 @@ def mkdir(path, overwrite=True):
+    family_is_vf(ttFonts)
+    family_names = set(font_familyname(f) for f in ttFonts)
+    if len(family_names) != 1:
+        raise ValueError(f"Multiple families found {family_names}")
+    return True
+
+
+def unique_name(ttFont, nameids):
+    font_version = _font_version(ttFont)
+    vendor = ttFont["OS/2"].achVendID.strip()
+    ps_name = nameids[6]
+    return f"{font_version};{vendor};{ps_name}"
+
+
+def _font_version(font, platEncLang=(3, 1, 0x409)):
+    nameRecord = font["name"].getName(5, *platEncLang)
+    if nameRecord is None:
+        return f'{font["head"].fontRevision:.3f}'
+    # "Version 1.101; ttfautohint (v1.8.1.43-b0c9)" --> "1.101"
+    # Also works fine with inputs "Version 1.101" or "1.101" etc
+    versionNumber = nameRecord.toUnicode().split(";")[0]
+    return versionNumber.lstrip("Version ").strip()
+
+
+def partition_cmap(font, test, report=True):
+  """Drops all cmap tables from the font which do not pass the supplied test.
+
+  Arguments:
+    font: A ``TTFont`` instance
+    test: A function which takes a cmap table and returns True if it should
+      be kept or False if it should be removed from the font.
+    report: Reports to stdout which tables were dropped and which were kept.
+
+  Returns two lists: a list of `fontTools.ttLib.tables._c_m_a_p.*` objects
+  which were kept in the font, and a list of those which were removed."""
+  keep = []
+  drop = []
+
+  for index, table in enumerate(font['cmap'].tables):
+    if test(table):
+      keep.append(table)
+    else:
+      drop.append(table)
+
+  if report:
+    for table in keep:
+        print(("Keeping format {} cmap subtable with Platform ID = {}"
+               " and Encoding ID = {}").format(table.format,
+                                               table.platformID,
+                                               table.platEncID))
+    for table in drop:
+        print(("--- Removed format {} cmap subtable with Platform ID = {}"
+             " and Encoding ID = {} ---").format(table.format,
+                                                 table.platformID,
+                                                 table.platEncID))
+
+  font['cmap'].tables = keep
+  return keep, drop
+
+
+def _unicode_marks(string):
+    unicodemap = [(u'©', '(c)'), (u'®', '(r)'), (u'™', '(tm)')]
+    return filter(lambda char: char[0] in string, unicodemap)
+
+
+def normalize_unicode_marks(string):
+    """ Converts special characters like copyright,
+        trademark signs to ascii name """
+    # print("input: '{}'".format(string))
+    input_string = string
+    for mark, ascii_repl in _unicode_marks(string):
+        string = string.replace(mark, ascii_repl)
+
+    rv = []
+#    for c in unicodedata.normalize('NFKC', smart_text(string)):
+    for c in unicodedata.normalize('NFKC', string):
+        # cat = unicodedata.category(c)[0]
+        # if cat in 'LN' or c in ok:
+        rv.append(c)
+
+    new = ''.join(rv).strip()
+    result = unidecode(new)
```

Comment:
This module hasn't imported this function.

Reply:
Fixed already in b7ba4!